src/main/java/slimeknights/tconstruct/shared/block/BlockGlow.java

```diff
@@ -51,10 +57,65 @@ public int getMetaFromState(IBlockState state) {
   public IBlockState getStateFromMeta(int meta) {
     return getDefaultState().withProperty(FACING, EnumFacing.getFront(meta));
   }
+  
+  @Override
+  public ItemStack getPickBlock(IBlockState state, RayTraceResult target, World world, BlockPos pos, EntityPlayer player) {
+    // use the glowball for the pickblock, as it is more useful than the technical block
+    if(gadgetsLoaded) {
```

Comment:
I'd reduce this to a simple `if(TinkerGadgets.throwball != null)`. That way there is no hard dependency on the pulse.